run

```diff
@@ -92,6 +95,10 @@ for input_file in $INPUT_DIR/file/*; do
     cp -a $input_file $INPUT_DIR
   fi
 
+  ##############################################################################
+  # Tag all of the files so that they are not re-reaped
+  python tagger.py "$dicom_input" -g '0x0021' -i 'Flywheel' -t 'DICOM Send'
```

Comment:
Highly recommend that you check your exit code here, and do **NOT** proceed with sending if tagging fails.